app/views/about_sections/_staff.html.erb

```diff
@@ -1,7 +1,38 @@
-<ul class="c-staff row">
-  <% users.each do |user| %>
-    <li class="small-6 medium-3 large-2 columns">
-      <%= render 'shared/user', user: user %>
-    </li>
-  <% end %>
-</ul>
+<%
+  categories = {}
+  users.each do |user|
+    if user.position_category.present?
+      categories[user.position_category] ||= []
+      categories[user.position_category].push(user)
+    end
+  end
+%>
+
+<div class="row">
+  <div class="small-12 column">
+    <ul class="c-staff-selector">
+      <% categories.each_with_index do |category, index| %>
```

Comment:
I think we could do:

`users.group_by{|t| t.position_category}.each_with_index do |category, index|`